litestar/cli/_utils.py

```diff
@@ -107,10 +107,13 @@ def from_env(cls, app_path: str | None, app_dir: Path | None = None) -> Litestar
 
             dotenv.load_dotenv()
         app_path = app_path or getenv("LITESTAR_APP")
+        app_name = getenv("LITESTAR_APP_NAME") or "Litestar"
+        quiet_console = getenv("LITESTAR_QUIET_CONSOLE") or False
         if app_path and getenv("LITESTAR_APP") is None:
             os.environ["LITESTAR_APP"] = app_path
         if app_path:
-            console.print(f"Using Litestar app from env: [bright_blue]{app_path!r}")
+            if not quiet_console:
+                console.print(f"Using {app_name} from env: [bright_blue]{app_path!r}")
```

Comment:
```suggestion
                console.print(f"Using {app_name} app from env: [bright_blue]{app_path!r}")
```

Reply:
I purposely removed this word to make the language read a bit more clearlty.